### PR TITLE
Create v0.3.0 Release notes

### PR DESCRIPTION
**Pull request by yaron2**
^ Title

### CODE REVIEW

File: docs/release_notes/v0.3.0.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+```
+helm upgrade dapr dapr/dapr --set global.tag=0.3.0
+```
+
+*Note: If you installed with a different chart name, use `helm upgrade <your-chart-name>`*
+
+## Dapr Runtime
+ 
+**Added** Couchbase as a State Store
```
Comment (LMWF) [inline docs/release_notes/v0.3.0.md]: Service discovery componentization was in this release.

Comment (yaron2) [reply docs/release_notes/v0.3.0.md]: Right! I wonder why I didn't see that as part of the project

Comment (yaron2) [reply docs/release_notes/v0.3.0.md]: Fixed

Comment (LMWF) [reply docs/release_notes/v0.3.0.md]: Came in right at the end of 0.2 but wasn't part of 0.2.  Probably the card got shuffled.

**Comment by yaron2** [comment]
Closing this as needs to be added to 0.3.0-rc branch first

**Pull request closed without merging**